src/client/dfuse/ops/lookup.c

```diff
@@ -61,13 +63,13 @@ dfuse_reply_entry(struct dfuse_projection_info *fs_handle,
 
 		/* Update the existing object with the new name/parent */
 
-		DFUSE_TRA_DEBUG(ie, "inode dfs %p %ld hi %#lx lo %#lx",
+		DFUSE_TRA_DEBUG(inode, "inode dfs %p %#lx hi %#lx lo %#lx",
 				inode->ie_dfs,
 				inode->ie_dfs->dfs_ino,
 				inode->ie_oid.hi,
 				inode->ie_oid.lo);
 
-		DFUSE_TRA_DEBUG(ie, "inode dfs %p %ld hi %#lx lo %#lx",
+ 		DFUSE_TRA_DEBUG(ie, "inode dfs %p %#lx hi %#lx lo %#lx",
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs